app/server/models/credit_transfer.py

```diff
@@ -301,6 +301,7 @@ def resolve_as_complete(self, batch_uuid=None):
 
```

Comment:
What was going on was that `self.check_sender_transfer_limits()` would resolve a transfer as rejected if a limit was reached, and that would set the blockchain status as UNSTARTED. But here (on line 297) we allow sempoadmins to exceed the limits, therefore we have to reset blockchain_status to PENDING if the limit check doesn't raise an exception.